metadata-ingestion/src/datahub/__init__.py

```diff
@@ -16,6 +19,38 @@ def nice_version_name() -> str:
     return __version__
 
 
+def git_tag(version: Optional[str] = None) -> str:
+    """
+    Attempt to extract the expected github tag based on environment or version. Fallback to `master`
+    :return: github tag
+    """
+    if version not in ("master", "head"):
+        match = re.match(
+            "^v?([0-9]+[.][0-9]+([.][0-9]+){1,2})",
+            os.getenv("DATAHUB_VERSION", version if version else __version__),
+        )
+        if match and match.group(1) != "0.0.0":
```

Comment:
Based on Paul's error this seems to not be working

Reply:
which is odd because it should be doing OK. It checks out fine here:
![image](https://user-images.githubusercontent.com/101486603/217110721-7691b2bd-92d9-4a9f-9571-b7e8292fb164.png)